app/frontend.py

```
@@ -3,48 +3,139 @@
-            st.write(f"Label: {'Suspicious ⚠️' if out['label'] else 'Safe ✅'}")
-            st.write(f"Phishing Probability: {out['phish_prob']:.2f}")
+# Add tabs for different analysis types
+tab1, tab2, tab3, tab4 = st.tabs(["📧 Text Email", "📁 EML Files", "🎵 Audio", "🔄 Combined"])
+
+with tab1:
+    st.header("Check Email Text")
+    st.write("Paste email content (loses hidden links and metadata)")
+    email_text = st.text_area("Paste email text here:")
+    if st.button("Check Email Text"):
+        if email_text:
+            resp = requests.post("http://127.0.0.1:8000/check_email", json={"text": email_text})
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)